lib/models/ui/challenge_details.dart

```diff
@@ -1,3 +1,4 @@
+import "package:cloud_firestore/cloud_firestore.dart";
 import "package:flutter/foundation.dart";
```

Comment:
Is there any way to avoid using cloud firestore in the UI? Like converting to `LatLng` before passing it to the UI.
We should keep our UI and model separated.

Reply:
That makes sense. I did that in 7f2b06bc608a461b67b348f8558446793cf54333